tests/framework/icd/test_icd.cpp

```diff
@@ -596,7 +605,21 @@ VKAPI_ATTR VkResult VKAPI_CALL test_vkGetPhysicalDeviceImageFormatProperties(VkP
 VKAPI_ATTR void VKAPI_CALL test_vkGetPhysicalDeviceFeatures2(VkPhysicalDevice physicalDevice,
                                                              VkPhysicalDeviceFeatures2* pFeatures) {}
 VKAPI_ATTR void VKAPI_CALL test_vkGetPhysicalDeviceProperties2(VkPhysicalDevice physicalDevice,
-                                                               VkPhysicalDeviceProperties2* pProperties) {}
+                                                               VkPhysicalDeviceProperties2* pProperties) {
+    if (pProperties) {
+        test_vkGetPhysicalDeviceProperties(physicalDevice, &pProperties->properties);
+        VkBaseInStructure* pNext = reinterpret_cast<VkBaseInStructure*>(pProperties->pNext);
+        while (pNext) {
+            if (pNext->sType == VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_PCI_BUS_INFO_PROPERTIES_EXT) {
+                auto& phys_dev = icd.GetPhysDevice(physicalDevice);
```

Comment:
I'd move this out of the if statement so future pNext chain filling structs won't have to copy/paste it.

Reply:
The phys_dev or something else?

Reply:
I moved the phys_dev out, so if that's what you meant, it's done.

Reply:
Yes, that is what I meant. Sorry for the confusion.